### PR TITLE
Simon Elhoej Steinmejer

### PR DESCRIPTION
**Pull request by Elhoej**

### CODE REVIEW

File: pointers/pointers.c [hunk 1]
```diff
@@ -21,7 +24,15 @@ void swap(int* a, int* b)
 */
 char *find_char(char *str, int c)
 {
-
+    for (int i = 0; i < strlen(*str); i++)
+    {
```
Comment (ericblancas23) [inline pointers/pointers.c]: Great job, glad you are getting through this just fine Simon :)

File: strings/strings.c [hunk 2]
```diff
@@ -20,7 +21,18 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
-
+    int length = strlen(s);
+    int end = length - 1;
+    int begin = 0;
+
+    for (begin = 0; begin < length; begin++)
```
Comment (ericblancas23) [inline strings/strings.c]: let me know if the tests are working for you, hopefully your code is passing just fine

File: fizzbuzz/fizzbuzz.c [hunk 3]
```diff
@@ -10,7 +10,25 @@
 */
 int fizzbuzz(int n)
 {
+    int counter = 0;
+    for(int i = 0; i <= n; i++)
+    {
+        if (i % 3 == 0 && i % 5 == 0)
+        {
+            printf("FizzBuzz\n");
+        } else if (i % 3 == 0)
```
Comment (ericblancas23) [inline fizzbuzz/fizzbuzz.c]: good start!, nothing like an easy fizzbuzz to get you started in C

File: structs/structs.c [hunk 4]
```diff
@@ -8,7 +8,10 @@
     Person should have the fields `name`, `age`, `height`, and `weight`.
 */
 typedef struct Person {
-
+    char name[20];
+    int age;
```
Comment (ericblancas23) [inline structs/structs.c]: Loving your progress Simon keep it up!, and let me know if you have any questions

File: strings/strings.c (hunk 2 above)
Comment (Elhoej) [reply strings/strings.c]: Tests are passing for me, are they not for you?

**Pull request closed without merging**